utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,144 @@
+        logger.debug("Decrypt API response status: %d", resp.status_code)
+        resp.raise_for_status()
+        
+        result = resp.json()           # e.g. [ {key1: plain1}, {key2: plain2} ]
+        logger.debug("Decrypt API returned %d result entries", len(result))
+        
+        out = {}
+        for entry in result:
+            out.update(entry)
+        
+        logger.info("Successfully decrypted %d fields: %s", len(out), list(out.keys()))
+        return out
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)